content/guides/monitoring.md

```diff
@@ -336,7 +336,50 @@ right people get notified so the problem can be resolved as soon as possible.
    anytime the monitor renotifies. The original message will be included as
    well.
 
-## Monitor FAQs 
+
+### Message template variables
+
+Message template variables can be used to customize your monitor notifications.
+This feature is supported in all monitor types. There are two primary use cases
+for template variables: 1) displaying a different message depending on the
+notification type (e.g. triggered, recovered, no data) and 2) incorporating the
+triggering scope into the message of Multi Alerts.
+
+1. **Conditional variables for different notification types**: You can have a
+    monitor event display a different message depending on whether the event is a
+    trigger, resolve, or no data notification. These variables use simple if-else
+    logic with the following syntax:
+
+        {{#var}} This text will show {{/var}} only if the variable `var` is true.
+        {{^var}} This text will show {{/var}} only if `var` is NOT true.
+
+    The conditional variables available are is_alert, is_recovery, and is_no_data.
+    These can also be seen in the "Use message template variables" help box in
+    Step 3 of the monitor editor.
+
+2. **Tag variables for Multi Alerts**: When your monitor is a Multi Alert, instead
+    of having a generic message (and finding the triggering tag scope in the alert
+    query definition), a variable can be used in the message for explicitly
+    identifying the triggering scope.
+
+    For instance, here is an example Multi Alert event notification that doesn't
+    use template variables:
+
+    ![no template var](/static/images/monitor/templatevar1.png)
+
+    and here is one that does (variable is `host.name`, highlighting added):
+
+    ![template var](/static/images/monitor/templatevar2.png)
+
+    The tag template variables available depend on the tag group selected in Step 1
+    of the monitor editor. The possible options will automatically populate at the
+    bottom of the "Use message template variables" help box in Step 3 of the editor.
+    These variables can also be used in the monitor titles (names), but note that
+    the variables are only populated in the text of Datadog child events (not the
+    parent, which displays an aggregation summary).
+
```

Comment:
Similarly it might be useful to show an example message here, to see how it translates. I'd even go a step further with your example and show how you can use it in follow-up steps, e.g.:
```
1. ssh to {{host.ip}}
2. Run command `fix_disk_space`
3. ...
```

I think we might have samples in our account...? Maybe even showcase using more than one, although that's not definitely necessary.